popper.el

```diff
@@ -257,13 +257,17 @@ grouped by the predicate `popper-group-function'.")
 
 (defun popper-select-popup-at-bottom (buffer &optional _alist)
   "Display and switch to popup-buffer BUFFER at the bottom of the screen."
-  (let ((window (display-buffer-in-side-window
-                 buffer
-                 `((window-height . ,popper-window-height)
-                   (side . bottom)
-                   (slot . 1)))))
+  (let ((window (popper-popup-at-bottom buffer _alist)))
     (select-window window)))
 
+(defun popper-popup-at-bottom (buffer &optional _alist)
```

Comment:
Rename to `popper-display-popup-at-bottom` (in keeping with the display-buffer function naming `display-buffer-*`)